### ISSUE TITLE
Improve tests

### ISSUE THREAD

**Issue by wb9688** [TeamNewPipe/NewPipeExtractor#266]
While implementing `yt_new`, I found out a lot of things just aren't tested at all, for example the InfoItemExtractors. I'll update this issue with more and concrete examples later.

**Comment by PeterHindes** [comment]
Yes, I noticed a lot of regressions in the new versions. It would be nice to have tests for all the possible null values. For example properly handling unlikely cases like no upload date.
